### PR TITLE
Add bounty title to notifications

### PR DESCRIPTION
**Pull request by c-o-l-o-r**
@villanuevawill

### CODE REVIEW

**Comment by villanuevawill** [comment]
@codeluggage want to have you review this as another means of diving in :)

File: bounties_api/notifications/notification_client.py [hunk 1]
```diff
@@ -283,6 +302,7 @@ def rating_received(self, bounty_id, event_date, uid, receiver, **kwargs):
             user=receiver,
             data=string_data,
             subject='You Received a New Rating on Your Bounty',
+            bounty_title=bounty.title,
```
Comment (villanuevawill) [inline bounties_api/notifications/notification_client.py]: @c-o-l-o-r what about for ratings?

Comment (villanuevawill) [reply bounties_api/notifications/notification_client.py]: Don't we need to send some more data in?

**Comment by villanuevawill** [comment]
2 things I see immediately - 
1. We have to send data for ratings
2. We need to update the function to store the user who the notification is coming from (The notifications bar has the img as an example). :)

**Comment by c-o-l-o-r** [comment]
@villanuevawill added a `from_user` field in the notifications model, so that should be all good. We decided to punt sending ratings data at the moment and contribution added is not sending the user who initiated the contribution since we'll need to refactor the contract listener flow a bit.

File: bounties_api/notifications/notification_client.py [hunk 2]
```diff
@@ -127,16 +141,20 @@ def fulfillment_updated(
             str(notifications['FulfillmentUpdatedIssuer']),
             notification_name=notifications['FulfillmentUpdatedIssuer'],
             user=bounty.user,
-            data=string_data_issuer,
-            subjet='Submission was Updated',
+            from_user=fulfillment.user,
+            string_data=string_data_issuer,
+            bounty_title=bounty.title,
+            subject='Submission was Updated',
```
Comment (codeluggage) [inline bounties_api/notifications/notification_client.py]: This wasn't part of your change, but it seems like weird capital casing. Is this intentional @villanuevawill ?

Comment (c-o-l-o-r) [reply bounties_api/notifications/notification_client.py]: @codeluggage i think the reasoning behind it is that the subject field is used to populate the subject field in an email that is sent out to the user--therefore, it is capitalized like a title

**Pull request merged**